include/rocksdb/cache.h

```diff
@@ -59,7 +59,7 @@ struct LRUCacheOptions {
   //
   // See also
   // BlockBasedTableOptions::cache_index_and_filter_blocks_with_high_priority.
-  double high_pri_pool_ratio = 0.0;
+  double high_pri_pool_ratio = 0.5;
```

Comment:
Since we are on this, it might be worth updating the comment above to be better. 
Right now, at least to me, the comment gives the impression that unless the user explicitly classifies things as high pri and low pri (say, by setting `cache_index_and_filter_blocks_with_high_priority`) this will not work. The users might not be aware that this will work even if all blocks are classified with the same priority.

Reply:
I guess you are right that the diff might not be effective at all... Let me take a look.

Reply:
I checked the code and it does seem to take effective even with cache_index_and_filter_blocks_with_high_priority=false.

Reply:
I changed cache_index_and_filter_blocks_with_high_priority default too.